### PR TITLE
+nvs.rm

### PR DESCRIPTION
**Pull request by elsbiet**
added nvs.rm(namespace, key) return: true|false

### CODE REVIEW

**Comment by elsbiet** [comment]
added nvs.rm(namespace, key) return: true|false

File: components/lua/modules/sys/nvs.c [hunk 1]
```diff
@@ -297,7 +300,7 @@ static int l_nvs_exists(lua_State *L) {
     }
 
     // Try to open
-    err = nvs_open(nspace, NVS_READWRITE, &handle_to_settings);
+    err = nvs_open(nspace, NVS_READONLY, &handle_to_settings);
```
Comment (elsbiet) [inline components/lua/modules/sys/nvs.c:303]: NVS_READWRITE -> NVS_READONLY

File: components/lua/modules/sys/nvs.c [hunk 2]
```diff
@@ -209,7 +211,7 @@ static int l_nvs_read(lua_State *L) {
     }
 
     // Open
-    err = nvs_open(nspace, NVS_READWRITE, &handle_to_settings);
+    err = nvs_open(nspace, NVS_READONLY, &handle_to_settings);
```
Comment (elsbiet) [inline components/lua/modules/sys/nvs.c:214]: NVS_READWRITE -> NVS_READONLY

**Comment by elsbiet** [comment]
@jolivepetrus, thanks

should i close this pull request now?

**Comment by thunderWhitecat** [comment]
@spellauge,

No, it is yet merged and closed.

**Comment by the0ne** [comment]
@jolivepetrus documented at https://github.com/the0ne/Lua-RTOS-ESP32/wiki/NVS-module

**Pull request merged**